### PR TITLE
[BTS-477] Added tests for log parameters

### PR DESCRIPTION
**Pull request by cpjulia**
### Scope & Purpose

This PR adds coverage tests for log parameters mentioned in https://arangodb.atlassian.net/wiki/spaces/AQA/pages/1492287509/Logging.

- [ ] :hankey: Bugfix
- [x] :pizza: New feature
- [ ] :fire: Performance improvement
- [ ] :hammer: Refactoring/simplification

### Checklist

- [x] Tests
  - [ ] **Regression tests**
  - [ ] C++ **Unit tests**
  - [x] **integration tests**
  - [ ] **resilience tests**
- [x] :book: CHANGELOG entry made
- [ ] :books: documentation written (release notes, API changes, ...)
- [ ] Backports (maybe not)

#### Related Information

*(Please reference tickets / specification / other PRs etc)*

- [ ] Docs PR: 
- [x] Enterprise PR: https://github.com/arangodb/enterprise/pull/1072
- [x] GitHub issue / Jira ticket: https://arangodb.atlassian.net/browse/BTS-477
- [ ] Design document:

### CODE REVIEW

File: tests/js/client/server_parameters/log-param-options-.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
...
+      assertTrue(res.hasOwnProperty("topic"));
+      assertTrue(res.hasOwnProperty("level"));
+      assertTrue(res.hasOwnProperty("timestamp"));
+      assertTrue(res.hasOwnProperty("text"));
+
+      for (let i = 0; i < res.totalAmount; ++i) {
+        assertMatch(/\d{1,}/, res.lid[i]);
+        assertMatch(/[A-Za-z]{1,}/, res.topic[i]);
+        assertMatch(/\d{1,}/, res.level[i]);
+        assertMatch(/\d{1,}/, res.timestamp[i]);
```
Comment (jsteemann) [inline tests/js/client/server_parameters/log-param-options-.js:111]: These regexes are a bit too loose.
For example, `\d{1,}` matches any string that contains at least one numeric digit, e.g. `42`, `998599004` but also `piffpaff9puff`. I suggest anchoring all the regexes using `^` and `$`.

File: tests/js/client/server_parameters/log-json-format-info-cluster.js [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+    },
+
+    tearDown: function() {
+      db._useDatabase("_system");
+      db._dropDatabase(dbName);
+    },
+
+    testLogClusterInfoInFile: function() {
+      let coordinatorEndpoint = getCoordinatorEndpoints()[0];
+      arango.reconnect(coordinatorEndpoint, db._name(), "root", "");
```
Comment (jsteemann) [inline tests/js/client/server_parameters/log-json-format-info-cluster.js:78]: this "reconnect" will potentially have a side effect that at before this test we may be connected to a different server than after this test.
If a test reconnects, it should make sure in its `tearDown` / `tearDownAll` that the pre-test state is restored when the test completes.

Comment (cpjulia) [reply tests/js/client/server_parameters/log-json-format-info-cluster.js:80]: Addressed

File: tests/js/client/server_parameters/log-param-options-.js (hunk 1 above)
Comment (cpjulia) [reply tests/js/client/server_parameters/log-param-options-.js]: Addressed

**Pull request merged**